src/main/java/mat/client/measure/AbstractNewMeasureView.java

```diff
@@ -2,6 +2,9 @@
 
 import java.util.List;
 
+import com.google.gwt.user.client.Window;
```

Comment:
I think this import was just here for testing and we shouldn't need it any more.

Reply:
that's right, will remove all unused imports. Thanks!